src/main/java/htsjdk/samtools/SRAFileReader.java

```diff
@@ -61,7 +61,8 @@ public SRAFileReader(final SRAAccession acc) {
         this.acc = acc;
 
         if (!acc.isValid()) {
-            throw new IllegalArgumentException("Invalid SRA accession was passed to SRA reader: " + acc);
+            throw new IllegalArgumentException("SRAFileReader: cannot resolve SRA accession '" + acc + "'\n" +
```

Comment:
isn't it clear at some point in the code that a connection has been established but that the accession is bad? why not inform the user?